gubernator/github/classifier.py

```diff
@@ -21,7 +21,7 @@
 import models
 
 
-XREF_RE = re.compile(r'k8s-gubernator.appspot.com/build(/[^])\s]+/\d+)')
+XREF_RE = re.compile(r'(?:k8s-gubernator\.appspot\.com|gubernator\.k8s\.io)/build(/[^])\s]+/\d+)')
```

Comment:
maybe also support gubernator.kubernetes.io? or maybe we should make that redirect to gubernator.k8s.io, prow-style?

Reply:
Didn't know about that other host. Redirecting sounds fine, but this is just for GitHub xrefs, and presumably bot comments will prefer gubernator.k8s.io over the other options?

Reply:
We'll probably prefer the shorter one, but gubernator is serving on all three hosts now.